### ISSUE TITLE
Admin_logs MemberKick addAdminLog error

### ISSUE THREAD

**Issue by Eccleria** [Titch88/ewibot#90]
```javascript
member kicked from/left Discord Server
memberKick User {
  id: '491648066948956190',
  bot: false,
  system: false,
  flags: UserFlags { bitfield: 256 },
  username: 'jojoledebile',
  discriminator: '4142',
  avatar: null,
  banner: undefined,
  accentColor: undefined
}
memberKick diff null
file:///home/ewibot/ewibot/src/helpers/dbHelper.js:8
  data[index].push(messageId);
              ^

TypeError: Cannot read properties of undefined (reading 'push')
    at addAdminLogs (file:///home/ewibot/ewibot/src/helpers/dbHelper.js:8:15)
    at file:///home/ewibot/ewibot/src/admin/listeners.js:790:7
    at Array.forEach (<anonymous>)
    at Client.onGuildMemberRemove (file:///home/ewibot/ewibot/src/admin/listeners.js:789:17)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by Eccleria** [comment]
Occurred once again

**Comment by Eccleria** [comment]
```javascript
member kicked from/left Discord Server
memberKick User {
  id: '574133134002618379',
  bot: false,
  system: false,
  flags: UserFlags { bitfield: 0 },
  username: 'Ellyndia (Ellyndia_Tempel)',
  discriminator: '6068',
  avatar: '8ebbe2ed7f68c5c5bf9c7b075218fa97',
  banner: undefined,
  accentColor: undefined
}
memberKick diff null
file:///home/ewibot/ewibot/src/helpers/dbHelper.js:7
  data[index].push(messageId);
              ^

TypeError: Cannot read properties of undefined (reading 'push')
    at addAdminLogs (file:///home/ewibot/ewibot/src/helpers/dbHelper.js:7:15)
    at file:///home/ewibot/ewibot/src/admin/listeners.js:830:7
    at Array.forEach (<anonymous>)
    at Client.onGuildMemberRemove (file:///home/ewibot/ewibot/src/admin/listeners.js:829:17)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by Eccleria** [comment]
fixed in #110
